composer.json

```diff
@@ -108,7 +108,8 @@
         "league/flysystem-aws-s3-v2": "Required to use the Flysystem S3 driver (~1.0).",
         "league/flysystem-rackspace": "Required to use the Flysystem Rackspace driver (~1.0).",
         "pda/pheanstalk": "Required to use the beanstalk queue driver (~3.0).",
-        "predis/predis": "Required to use the redis cache and queue drivers (~1.0)."
+        "predis/predis": "Required to use the redis cache and queue drivers (~1.0).",
+        "pubnub/pubnub": "Required to use the PubNub event broadcast driver (~3.7)."
```

Comment:
Ping @cwt137 You need to fix merge conflicts, and you need to add and entry to https://github.com/laravel/framework/blob/master/src/Illuminate/Broadcasting/composer.json#L32.